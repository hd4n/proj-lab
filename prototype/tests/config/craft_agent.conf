#item_nev id
#equipment, material
@items
am a1
am a2
am a3
nu n1
nu n2
nu n3

#effekt_nev id duration
@effects
fo f1 1

#agent_nev id effekt_id
@agents
vi vi1 f1

#kod_nev id agent_id aminocost nucleocost
@codes
co cd1 vi1 3 3

#mezo_nev id [tartalom]
@fields
em e1

#id akt_mezo [item_id1;item_id2...] [material_id1;material_id2...] [effect_id1;effect_id2...]
@virologists
v1 e1 null a1;a2;a3;n1;n2;n3 null