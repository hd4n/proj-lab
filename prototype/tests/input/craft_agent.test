load craft_agent.conf
getstats v1
craft v1 cd1
getstats v1
