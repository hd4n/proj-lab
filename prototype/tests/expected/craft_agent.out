Load: sikeres

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_-1
	neededNucleo_-1
	co_null
	ma_a1+a2+a3+n1+n2+n3
	ag_null
	eq_null
	eqa_null
	ef_null
craft: v1 cd1

stats: v1
	fi_e1
	di_null
	st_false
	res_0.0
	re_false
	ma_10
	ref_0
	neededAmino_0
	neededNucleo_0
	co_null
	ma_a2+a3+n2+n3
	ag_vi1
	eq_null
	eqa_null
	ef_null
